### ISSUE TITLE
Two problems in tests phase

### ISSUE THREAD

**Issue by Universebenzene** [PennyQ/astro-vispy#374]
**Describe the bug**

1. Tests cannot begin with vispy>=0.9.1, as it gives:
```
E   ImportError: cannot import name 'VERT_SHADER' from 'vispy.visuals.volume' (/usr/lib/python3.10/site-packages/vispy/visuals/volume.py)
```
Also the `import glue_vispy_viewers.isosurface.multi_iso_visual` will give the same error.

2. When test with vispy 0.9.0, tests in `test_session_back_compat.py` will give `ModuleNotFoundError: No module named 'echo.list'`. Probably because [the commit b809f7e](https://github.com/glue-viz/glue-vispy-viewers/commit/b809f7e74d4f92f8442b04d99ee38e2d33da5118) forgot to delete `list` after changed the `glue.external.echo` into `echo`, as the [glue.external.echo](https://github.com/glue-viz/glue/tree/v1.5.0/glue/external/echo) has `list.py`, but [echo](https://github.com/glue-viz/echo/tree/v0.5/echo) doesn't. Change all `echo.list.CallbackList` into `echo.CallbackList` or `echo.containers.CallbackList` will solve this problem.

**To Reproduce**

1. Run `import glue_vispy_viewers.isosurface.multi_iso_visual` or just run the tests with vispy >= 0.9.1 will get problem 1.
2. Runing tests with vispy 0.9.0 will get problem 2.

**Details:**

- Operating System: Arch Linux
- Python version (``python --version``): Python 3.10.5
- Glue version (``glue --version``): 1.5.0
- How you installed glue: install the package built via [PKGBUILD](http://fars.ee/2T6C/sh) on archlinux, with`python setup.py` command inside the packaging script.

**Additional context**

I also tried running `import glue_vispy_viewers.isosurface.multi_iso_visual` inside a virtualenv with glue-vispy-viewers installed via pip, and problem 1 occured as well.

**Comment by dhomeier** [comment]
Thanks for the report; the tests and some other issues are fixed in #373.
If you are familiar with installing dev versions, could you test if
`pip install --upgrade git+https://github.com/dhomeier/glue-vispy-viewers.git@78e0cc7`
works for you?
(Or checkout and install)
> git checkout -b dhomeier-echo-vispy-updates master
git pull https://github.com/dhomeier/glue-vispy-viewers.git echo-vispy-updates

This will probably still leave the isosurface renderer broken; I am afraid there is no short-term fix for that in sight.

**Comment by Universebenzene** [comment]
@dhomeier Ah, I forgot to check the on-going pull requests, so I didn't recognize your fixing 😂

I just built and installed a 1.0.5.r5.g78e0cc7 version from your repo, and neither of those two problems occured again. Good job 👍

Hope your fixing will be merged soon. Thanks! 😃

**Comment by dhomeier** [comment]
Closed by #373
